mars_api/data_import/data_importer.py

```diff
@@ -0,0 +1,134 @@
+from typing import Dict
+
+import colorlog
+from dateutil import parser
+from django.db.utils import IntegrityError
+
+from mars_api.serializers import GameSerializerForImportedData, PlayerScoreSerializer
+
+
+def str_to_bool(arg):
+    return bool(int(arg))
+
+
+GAME_FIELD_MAPPING = {
+    (0, "id", int),
+    (1, "date", parser.parse),  # datetime
+    (2, "game_map", str),
+    (3, "prelude", str_to_bool),
+    (4, "colonies", str_to_bool),
+    (5, "number_of_generations", int),
+}
+
+GAME_VALUE_REMAPPING = {"Default": "Tharsis"}
+
+PLAYER_SCORE_FIELD_MAPPING = {
+    (7, "player_nickname", str),
+    (8, "corporation", str),
+    (10, "terraform_rating", int),
+    (11, "milestones", int),
+    (12, "awards", int),
+    (13, "greeneries", int),
+    (14, "cities", int),
+    (15, "event_cards", int),
+    (16, "automated_cards", int),
+    (17, "active_cards", int),
+    (18, "resources", int),
+}
+
+PLAYER_SCORE_VALUE_REMAPPING = {
+    "UNMI": "United Nations Mars Initiative",
+    "Interpl. Cinematics": "Interplanetary Cinematics",
+}
+
+
+def remap_values(data: Dict[str, any], remapping: Dict[any, any]):
+    return {
+        key: (
+            remapping[value]
+            if not isinstance(value, list)
+            and not isinstance(value, dict)
```

Comment:
This loops 7K+ times and executes 20K+ conditional checks for ~450 rows of data

Reply:
I'm sure this could be done in 450(# of scores) + 160(# of games?) iterations without a single conditional check